website/docs/reference/resource-configs/dbt_valid_to_current.md

```diff
@@ -0,0 +1,96 @@
+---
+resource_types: [snapshots]
+description: "Snapshot dbt_valid_to_current custom date"
+datatype: "{<dictionary>}"
+default_value: {"dbt_valid_from": "dbt_valid_from", "dbt_valid_to": "dbt_valid_to", "dbt_scd_id": "dbt_scd_id", "dbt_updated_at": "dbt_updated_at"}
```

Comment:
The default value for `dbt_valid_to_current` is `NULL`.

I'm guessing this was a copy-paste carry-over from the docs for `snapshot_meta_column_names`.

Reply:
good shout!